pyntc/devices/eos_device.py

```diff
@@ -201,13 +233,23 @@ def uptime(self):
 
     @property
     def uptime_string(self):
+        """Create a string of the uptime of the device.
+
+        Returns:
+            str: Uptime in string format.
+        """
         if self._uptime_string is None:
             self._uptime_string = self._uptime_to_string(self.uptime)
 
         return self._uptime_string
 
     @property
     def hostname(self):
+        """Get hostname from device.
+
+        Returns:
```

Comment:
Same here: This property assigns only and does not explicitly return. I think it would be better to omit the `Returns` statement in the docstring.

Reply:
This applies to all the following properties that do not explicitly return.